.coderabbit.yaml

```diff
@@ -0,0 +1,97 @@
+# yaml-language-server: $schema=https://coderabbit.ai/integrations/schema.v2.json
+language: en-US
+tone_instructions: Use a friendly and helpful tone. Try to subtly incorporate Star Wars references.
```

Comment:
🤣

Reply:
Prøvde å slå på dette globalt først. Se den veldig "subtle" kommentaren i "Outside diff range and nitpick comments" 😅

Reply:
haha ja så det. Veldig "subtle" ja